samples/matter/light_switch/CMakeLists.txt

```diff
@@ -29,14 +29,9 @@ target_include_directories(app PRIVATE
     ${ZAP_PARENT_DIR}
 )
 
-target_compile_options(app PRIVATE -Wno-deprecated-declarations)
-
 target_sources(app PRIVATE
     src/app_task.cpp
-    src/light_switch.cpp
```

Comment:
can we remove those sources?

Reply:
Oh, I see my terrible mistake.. I used my script for aligning all samples on the CMakeLists.txt file as well.. Thanks for spotting that! :)